src/utils/source.js

```diff
@@ -235,7 +240,7 @@ function getMode(source: Source, sourceMetaData: SourceMetaDataType) {
 
   if (
     (url && url.match(/\.jsx$/i)) ||
-    (sourceMetaData && sourceMetaData.isReactComponent)
+    (sourceMetaData && (sourceMetaData.isReactComponent || symbols.hasJsx))
```

Comment:
lets simplify this:

`sourceMetaData && (sourceMetaData.isReactComponent || symbols.hasJsx)` => `symbols.hasJsx`

Reply:
I've got "invalid left-hand side in arrow function parameters" and some token errors. I've tried to fix that but nothing